exercises/querying-workflows/solution/signalclient.py

```diff
@@ -0,0 +1,14 @@
+import asyncio
+import time
+
+from temporalio.client import Client
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_workflow_handle("queries")
+    await handle.signal("submit_greeting", "User 1")
```

Comment:
```suggestion
    await handle.signal(Workflow.submit_greeting, "User 1")
```

Encourage the type safe form (same for exit below)